src/ol/format/KML.js

```diff
@@ -879,6 +894,71 @@ class KML extends XMLFeature {
     return regions;
   }
 
+  /**
+   * Read the cameras of the KML.
+   *
+   * @param {Document|Element|string} source Source.
+   * @return {Array<Object>} Cameras.
```

Comment:
Since this is part of the official API, I think the type should be more specific. Would you be able to add a `KMLCamera` typedef and use `{Array<KMLCamera>}` as return type?

Reply:
Done. I guess this should be done for other KML methods as well (e.g. region), but I do not have the time to spare atm